cli/cmd/tap.go

```diff
@@ -242,7 +242,16 @@ func writeTapEventsToBuffer(tapClient pb.Api_TapByResourceClient, w *tabwriter.W
 
 func renderTapEvent(event *common.TapEvent) string {
 	dst := util.AddressToString(event.GetDestination())
-	dstPod := event.GetDestinationMeta().GetLabels()["pod"]
+	dstLabels := event.GetDestinationMeta().GetLabels()
+	dstPod := dstLabels["pod"]
+	isSecured := "no"
+	controllerNs := dstLabels["conduit_io_control_plane_ns"]
+	controllerDstNs := dstLabels["dst_conduit_io_control_plane_ns"]
+
+	if controllerNs != "" && controllerNs == controllerDstNs {
```

Comment:
per your PR description, mind adding a `TODO` here (and perhaps reference an issue?) re: adding a `meshed`/`secured`/whatever label to the tap event?

Reply:
yeah!

Reply:
I could just switch this to use the 'meshed' label like the web and stat CLI does, that way everything gets converted once that label starts appearing... wdyt?

Reply:
yeah i think that sounds good, keep web/stat/tap all working the same way.